.github/workflows/multi-arch_compliance_operator_build.yml

```diff
@@ -0,0 +1,76 @@
+name: Multiarch build
+
+on:
+  push:
```

Comment:
hmm shouldn't this specify a branch?

Reply:
> 
It will not require branch name.
Also it is having on push mechanism so it will get triggered when it will get pushed.

Reply:
In our existing github workflows we have pushes based on "v*" and "main" branches, so we should do the same here.

Reply:
Added required changes.

Reply:
> In our existing github workflows we have pushes based on "v*" and "main" branches, so we should do the same here.

This is to prevent unnecessary actions for branches that are for active PRs.